test/toolbox_image.sh

```diff
@@ -545,23 +252,11 @@ run_test test_execute_mkfs_f2fs
 run_test test_execute_resizef2fs
 run_test test_execute_mount
 run_test test_execute_rsync
-run_test test_system_update_entrypoint
+run_test test_start_update_command
+run_test test_prepare_disk_command
+run_test test_update_files_command
```

Comment:
we may want to do this smarter, something like
```sh
test_script_command()
{
    for script in scripts/0?*.sh; do
        if [ ! -x "${toolbox_root_dir}${script}" ]; then
            echo "Warning: script '$script' not part of update toolbox, skipping'.
            continue
        fi
        chroot "${toolbox_root_dir}${script}" "-h"
    done
}
```

this does require us to define an interface for each script (every script must have the -h parameter, which is not a bad interface to agree upon).

What do you think?

Reply:
hmm, yeah that is true, it would have been nice to put them all under a simple template/define, hence the suggestion.
We also do this for the COMMANDS.

I'm not opposed to add 1 test per command, it _is_ good practice. but then it becomes a rather small step to also make each test a single file (or related tests together) and have one step in your gitlab-ci.yml file :)